op-devstack/sysgo/add_game_type.go

```diff
@@ -0,0 +1,111 @@
+package sysgo
+
+import (
+	"fmt"
+	"math/big"
+	"net/url"
+	"path"
+	"runtime"
+
+	"github.com/ethereum-optimism/optimism/op-chain-ops/devkeys"
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/artifacts"
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/manage"
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/stack"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/wait"
+	op_service "github.com/ethereum-optimism/optimism/op-service"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+func WithGameTypeAdded(l1ChainID eth.ChainID, l2ChainID eth.ChainID, l1ELID stack.L1ELNodeID) stack.Option[*Orchestrator] {
```

Comment:
Do we need both `l1ChainId` and `l1ELID`?  I think you can get the chainId from `l1ELID.`
```suggestion
func WithGameTypeAdded(l1ELID stack.L1ELNodeID l2ChainID eth.ChainID) stack.Option[*Orchestrator] {
```

Reply:
thanks. fixed.